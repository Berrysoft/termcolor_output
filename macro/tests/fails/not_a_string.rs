use termcolor_output as tco;

fn main() {
    let writer: Vec<u8> = vec![];
    let not_a_string = 0u32;
    tco::write!(&mut writer, not_a_string);
}
