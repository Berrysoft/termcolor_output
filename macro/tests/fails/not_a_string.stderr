error: The second argument must be a literal string
 --> $DIR/not_a_string.rs:6:27
  |
5 | |     let not_a_string = 0u32;
  | |_____________________________________________^
6 |       colored!(&mut writer, not_a_string);
  |  ___________________________^
